games/rust/entrypoint.sh

```diff
@@ -11,17 +11,36 @@ export INTERNAL_IP=`ip route get 1 | awk '{print $(NF-2);exit}'`
 MODIFIED_STARTUP=`eval echo $(echo ${STARTUP} | sed -e 's/{{/${/g' -e 's/}}/}/g')`
 echo ":/home/container$ ${MODIFIED_STARTUP}"
 
-# OxideMod has been replaced with uMod
-if [ -f OXIDE_FLAG ] || [ "${OXIDE}" = 1 ] || [ "${UMOD}" = 1 ]; then
+if [ -f CARBON_FLAG ] || [ "${FRAMEWORK}" == "carbon" ]; then
+    # Carbon: https://github.com/CarbonCommunity/Carbon.Core
+    echo "Updating Carbon..."
+    curl -sSL "https://github.com/CarbonCommunity/Carbon.Core/releases/download/production_build/Carbon.Linux.Release.tar.gz" | tar zx
+    echo "Done updating Carbon!"
+
+    # backward compatibility
+    export CARBON=1
+
+    export DOORSTOP_ENABLED=1
+    export DOORSTOP_TARGET_ASSEMBLY="$(pwd)/carbon/managed/Carbon.Preloader.dll"
+    MODIFIED_STARTUP="LD_PRELOAD=$(pwd)/libdoorstop.so ${MODIFIED_STARTUP}"
+
+elif [ -f OXIDE_FLAG ] || [ "${FRAMEWORK}" == "oxide" ]; then
+    # Oxide: https://github.com/OxideMod/Oxide.Rust
     echo "Updating uMod..."
     curl -sSL "https://github.com/OxideMod/Oxide.Rust/releases/latest/download/Oxide.Rust-linux.zip" > umod.zip
     unzip -o -q umod.zip
     rm umod.zip
     echo "Done updating uMod!"
+
+    # backward compatibility
+    export OXIDE=1
```

Comment:
There's no reason to export these values. All you need is the check above

Reply:
OK I understood the problem was with the downstream.